editor/editor_asset_installer.cpp

```diff
@@ -237,7 +237,7 @@ void EditorAssetInstaller::ok_pressed() {
 		//get filename
 		unz_file_info info;
 		char fname[16384];
-		ret = unzGetCurrentFileInfo(pkg, &info, fname, 16384, NULL, 0, NULL, 0);
```

Comment:
These were used to silence warnings: https://github.com/godotengine/godot/pull/10340#discussion_r133761341